Refactor app.js

isxwor opened the pull request:
Hello! I’ve removed unnecessary logics and refactored app.js and fixed a warning.

isxwor committed:
refactor app.js

isxwor committed:
fix warning

Adyasha8105 commented:
> Hello! I’ve removed unnecessary logics and refactored app.js and fixed a warning.

Thanks for the refactor @greyhere :D

isxwor commented:
> > Hello! I’ve removed unnecessary logics and refactored app.js and fixed a warning.
> 
> 
> 
> Thanks for the refactor @greyhere :D

You are welcome. :)
